tests/unit/services/resource-service.spec.jsx

```diff
@@ -156,32 +156,68 @@ describe('resourseService tests', () => {
     expect(response).toEqual(mockResponse)
   })
 
-   it('should edit an attachment', async () => {
-     const attachmentId = '6255bc080a75adf9223df444'
-     const attachment = {
-       description: 'Modified description',
-       category: '8655bc080a75adf9223df444'
-     }
-     const mockAttachmentResponse = {
-       ...attachment,
-       _id: attachmentId,
-       link: '1722535882408-test.pdf',
-       size: 15069,
-       resourceType: 'Attachment'
-     }
-
-     mockAxiosClient.onPatch().reply((config) => {
-       expect(config.url).toBe(`/attachments/${attachmentId}`)
-
-       return [200, mockAttachmentResponse]
-     })
-
-     const updatedAttachmentResponse =
-       await ResourceService.updateAttachmentQuery({
-         ...attachment,
-         id: attachmentId
-       })
-
-     expect(updatedAttachmentResponse).toEqual(mockAttachmentResponse)
-   })
+  it('should edit an attachment', async () => {
+    const attachmentId = '6255bc080a75adf9223df444'
+    const attachment = {
+      description: 'Modified description',
+      category: '8655bc080a75adf9223df444'
+    }
+    const mockAttachmentResponse = {
+      ...attachment,
+      _id: attachmentId,
+      link: '1722535882408-test.pdf',
+      size: 15069,
+      resourceType: 'Attachment'
+    }
+
+    mockAxiosClient
+      .onPatch(RegExp(URLs.resources.attachments.patch.replace(':id', attachmentId)))
```

Comment:
You should add `RegExp` only when you have `searchParameters` in the URL because mocked `axiosClient` expects a URL with `searchParameters`, so you should add regex just to not mock those parameters. Otherwise - just remove `RegExp` and check if the test works without it.
```suggestion
      .onPatch(URLs.resources.attachments.patch.replace(':id', attachmentId))
```